content/resources/cab_package/_index.md

```diff
@@ -1,35 +1,38 @@
 ---
 title: cab_package resource
 resource: cab_package
-draft: false
```

Comment:
I'm confused about this, was this intentional?

Reply:
That was probably not intentional. @IanMadd does that have a default value?

Reply:
It should be good. The default value is false.

Reply:
Cool we can stop generating that then. Is there a complete list of the defaults?